.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,22 @@
+
+name: Update CONTRIBUTORS file
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+jobs:
+  main:
+    runs-on: ubuntu-latest
```

Comment:
Please also add this line in here.
https://github.com/aws/aws-cdk/blob/main/.github/workflows/repo-metrics.yml#L14